railgun/Makefile

```diff
@@ -160,6 +160,19 @@ test.integration:
 	@./scripts/setup-integration-tests.sh
 	@GOFLAGS="-tags=integration_tests" go test -race -v ./test/integration/
 
+# Our integration tests, with verbose output
+.PHONY: test.integration
+test.integration:
+	@./scripts/setup-integration-tests.sh
+	@GOFLAGS="-tags=integration_tests" go test -race -v ./test/integration/
+
+# Our integration tests using the postgres backend, with verbose output
+# FIXME: race checking has been turned off because of race conditions found with deck (NEED TO POST ISSUE HERE)
```

Comment:
posting this comment just as a friendly reminder about `(NEED TO POST ISSUE HERE)` :)

Reply:
https://github.com/Kong/kubernetes-ingress-controller/runs/2576180289?check_suite_focus=true#step:5:669 appears to be the run that observed it, but I don't know the repro details yet.

Reply:
Going to follow up on this once the fix is released in Deck:

```suggestion
# TODO: race checking has been turned off because of race conditions found with deck
#       See: https://github.com/Kong/kubernetes-ingress-controller/issues/1324
```

Reply:
Oh boy. Is there a place where I can make bets on "temporary workarounds" being (or not) fixed within the promised timeframe :joy:? That will make me rich without the need to work.